### PR TITLE
Feature/add national service option

### PR DESCRIPTION
**Pull request by appsol**
### Add in a new "national" service option

Ref: https://trello.com/c/fDs5TiSH

- Added National field to create, edit and show views
- Added National filter to index search
- Updated Service show view to display Locations tab only if not a national service

### CODE REVIEW

File: src/views/services/forms/DetailsTab.vue [hunk 1]
```diff
@@ -88,6 +88,16 @@
           </template>
         </ck-image-input>
 
+        <ck-radio-input
+          :value="is_national"
+          @input="$emit('update:is_national', $event); $emit('clear', 'is_national')"
+          id="is_national"
+          :label="`National?`"
+          :hint="`Details if the service is only available at specific locations or can be accessed nationwide, e.g. via web or phone.`"
```
Comment (matthew-inamdar) [inline src/views/services/forms/DetailsTab.vue:96]: No need to use interpolation here.

```suggestion
          label="National?"
          hint="Details if the service is only available at specific locations or can be accessed nationwide, e.g. via web or phone."
```

**Review by matthew-inamdar — CHANGES_REQUESTED** [review]
Looks great 👍 

Minor comment to look at, but this looks good to go!

File: src/views/services/forms/DetailsTab.vue (hunk 1 above)
Comment (appsol) [reply src/views/services/forms/DetailsTab.vue]: Done.

**Review by matthew-inamdar — CHANGES_REQUESTED** [review]
The code looks good. I did some manual testing, and when you create a service, it takes you to the `Created` page. This page asks the user to then add a service location or to go to the service page. 

Since we can't add service locations to a national service, we should instead forward the user to the service page for a newly created national service.

We would still show the `Created` page for non-national services.

**Comment by appsol** [comment]
Changes completed

**Review by matthew-inamdar — APPROVED** [review]

**Pull request merged**